courses/free-courses-ru.md

```diff
@@ -136,6 +136,7 @@ ADV - Продвинутый. Тонкости.
 
 ### Python
 
+* [Автоматизация тестирования с помощью Selenium и Python](https://stepik.org/course/575/promo) - Stepik (INT)
```

Comment:
- A promo? 🤔 Could you give feedback?
- Maybe use top most level link: https://stepik.org/course/575
- Remember give credits to instructors and mention signup-up as notes acording to CONTRIBUTING guidelines